manifest.json

```diff
@@ -262,7 +278,10 @@
               "dockerfile": "2.0/jessie/kitchensink",
               "os": "linux",
               "tags": {
-                "1.0-2.0-2017-08-jessie": {},
+                "1.0-2.0-jessie": {},
```

Comment:
Looks like this is duplicated - see line 285.

Reply:
yup, good catch. Will remove